### PR TITLE
Paris logement famille

### PR DESCRIPTION
**Pull request by fpagnoux**
Connected to sgmap/mes-aides-ui#109

### CODE REVIEW

**Comment by MattiSG** [comment]
There's no test for `parisien`?

File: lib/models/situation.js [hunk 1]
```diff
@@ -164,6 +164,7 @@ var LogementDef = {
     conjointMemeAdresse: Boolean,
     loyer: Number,
     adresse: AdresseDef,
+    threeOutOfFiveYears: Boolean,
```
Comment (MattiSG) [inline lib/models/situation.js]: I don't find this name explicit enough. What do you think? What about something like `inhabitantForThreeYearsOutOfFiveLast`? It's ugly, but this is so specific the varname will likely have to be as specific…

**Comment by fpagnoux** [comment]
Done for the renaming. As we discussed, testing `parisien` doesn't seem relevant.

File: lib/config/ludwig.js [hunk 2]
```diff
@@ -134,6 +134,15 @@ module.exports = [
...
         'link':'http://www.education.gouv.fr/cid151/aides-financieres-au-lycee.html#Les%20bourses%20de%20lycée',
         'isBaseRessourcesYearMoins2':true,
-        'isSimulated': false
+    },
+    {
+        'id':'paris_logement_familles',
+        'isMontantAnnuel':false,
+        'label':'Paris Logement Familles',
+        'shortLabel':'PLF',
+        'description':'L’allocation Paris Logement Familles est versée, sous condition de ressources, aux familles parisiennes ayant au moins deux enfants ou un enfant handicapé à charge fiscale, afin de les aider à mieux supporter leurs dépenses de logement.',
```
Comment (MattiSG) [inline lib/config/ludwig.js]: L’allocation Paris Logement Familles permet aux familles parisiennes de mieux supporter leurs dépenses de logement.

**Comment by MattiSG** [comment]
GTM if only the description text is changed.

**Comment by MattiSG** [comment]
GTM

**Pull request merged**